### ISSUE TITLE
ideas of dev

### ISSUE THREAD

**Issue by martinbruno90** [wekan/wekan#1618]
this is not an issue but a proposition for dev:

I use wekan very often that is the best tool I ever used.  I have several boards and I have actions in some boards.  I would like to have a "summary" of all the actions I got to do and only mine, something like "mywekan", with all the contribution that I have to do.

Another proposition could be this, in the actions lists, could it be possible to define a leader for the action and a deadline for the action (not only for the card).

another idea could be to put all the actions in a GANTT planning or a calendar.

**Comment by xet7** [comment]
@martinbruno90 

1) Please make screenshot mockup about what kind of summary you do mean.

2) Please make screenshot mockup how it would work like in different steps.

3) There in Calendar issue #808 in progress, there received/start/due/end dates will be shown in calendar. If you mean something different, please comment there at #808

**Comment by martinbruno90** [comment]
[wekan_plus.pdf](https://github.com/wekan/wekan/files/1967362/wekan_plus.pdf)

**Comment by martinbruno90** [comment]
you can find here a short mockup of the project

**Comment by xet7** [comment]
@martinbruno90 

Are you running old version of Wekan? There is no due dates etc in mockup above. Can you upgrade to newest Wekan?

**Comment by martinbruno90** [comment]
I did not put due date to the card, but I would like to add due date to actions, into action lists.  No my wekan is uptodate :-)

**Comment by xet7** [comment]
@martinbruno90 

Status of card can be added with for example green color label with text "Done".

Linked card features are not yet merged to Wekan, you can [see screenshots at Linked card pull request](https://github.com/wekan/wekan/pull/1592). I need to test how well it works.

**Comment by martinbruno90** [comment]
yes I know and I use this very frequently.  Just to give an idea of the use of wekan here, I give you a screenshot of my boards :-)
[wekan_boards.pdf](https://github.com/wekan/wekan/files/1967580/wekan_boards.pdf)

**Comment by xet7** [comment]
@martinbruno90 

Latest version is v0.93, released today.
https://github.com/wekan/wekan/releases
